src/main/resources/org/jenkinsci/plugins/zap/ZAPDriver/config.jelly

```diff
@@ -128,6 +135,13 @@ SOFTWARE.
             <f:entry title="${%Logged in Indicator}" field="loggedInIndicator">
               <f:textbox clazz="required" />
             </f:entry>
+            <f:entry>
+            <f:optionalBlock title="${%Logged out Indicator}" field="activeLoggedOutIndicator" inline="true">
```

Comment:
No need for optional block, simply make it

             <f:entry title="${%Logged out Indicator}" field="loggedOutIndicator">
               <f:textbox />
             </f:entry>

The text box should also not make it required.